Dockerfile-alpine

```diff
@@ -70,18 +71,16 @@ RUN apk update && \
       php7-xdebug \
       php7-xml \
       php7-xmlreader \
+      php7-xmlwriter \
       php7-xsl \
       php7-zip \
       php7-zlib \
     && \
-    # Alpine + Ubuntu use different versioned names --> now standardized \
-    ln -s /usr/bin/php7 /usr/bin/php && \
-    ln -s /usr/sbin/php-fpm7 /usr/sbin/php-fpm7.0 && \
     # Disable xdebug by default \
     sed -i 's/zend_extension\s\?=/;zend_extension =/' $CONF_PHPMODS/xdebug.ini && \
     # Disable postgres by default \
-    sed -i 's/extension\s\?=/;extension =/' $CONF_PHPMODS/00_pdo_pgsql.ini && \
-    sed -i 's/extension\s\?=/;extension =/' $CONF_PHPMODS/00_pgsql.ini && \
+    sed -i 's/extension/;extension/' $CONF_PHPMODS/01_pdo_pgsql.ini && \
```

Comment:
Ah, in that case, keep the current regex.  Hadn't thought of the whitespace possibility.